spyderlib/qt/icon_manager.py

```diff
@@ -0,0 +1,218 @@
+import os
```

Comment:
I don't know if this file should be part of `spyderlib/qt` because this is just a module like `qtpy` (and soon to be replaced by `qtpy`! :-)

@goanpeca, what do you think?